app/javascript/components/ResumeGame.js

```diff
@@ -1,70 +1,73 @@
-import React, { Component } from 'react'
+import React, { Component } from "react";
 
-import './game/game.css'
-import AxiosInstance from '../actions/gameActions'
+import "./game/game.css";
+import AxiosInstance from "../actions/gameActions";
 
+class ResumeGame extends Component {
+  state = {
+    id: "",
+  };
 
+  onInputChange = ({ target: { name, value } }) => {
+    this.setState((prevState) => ({
+      ...prevState,
+      [name]: value,
+    }));
+  };
 
-class ResumeGame extends Component{
-    state = {
-            id:"",
-      }
+  resumeGameHandler = (e) => {
+    e.preventDefault();
+    AxiosInstance({ path: `games/${this.state.id}`, method: "get" })
+      .then((res) => {
+        this.props.resumeGame(res);
+      })
+      .catch(() => {});
+  };
 
-    onInputChange = ({ target: { name, value } }) => {
-        this.setState(prevState => ({
-          ...prevState,
-            [name]: value
-        }));
-      };
+  resumeLastGameHandler = () => {
+    const id = localStorage.getItem("gameId");
```

Comment:
`localStorage` is a hard dependency on this component, could we inject it so that it is easier to test?